docker-compose.yml

```
@@ -24,6 +24,8 @@ services:
     networks:
       - net
     env_file: sublime.env
+    environment:
+      MANTIS_HOST_URL: "http://sublime_mantis:8000"
```

actually the correct one for `dev`

```suggestion
      MANTIS_HOST_URL: "http://host.docker.internal:8000"
```